contracts/multisig/Cargo.toml

```diff
@@ -46,6 +46,7 @@ cosmwasm-schema = { workspace = true }
 cosmwasm-std = { workspace = true }
 cosmwasm-storage = { workspace = true }
 cw-storage-plus = { workspace = true }
+ed25519-dalek = { version = "2.1.1", default-features = false }
```

Comment:
Where's this used?

Reply:
https://github.com/axelarnetwork/axelar-amplifier/pull/284/files#diff-806ddeb05a6a238a6b153a52040776eef585c519227925ec480d7e91d42a10e7R227